consensus/src/state_computer.rs

```diff
@@ -76,21 +75,26 @@ impl LogicalTime {
     }
 }
 
+#[derive(Clone)]
+struct ExecutionProxyInner {
```

Comment:
Instead of`ExecutionProxyInner`, how about calling it `MutableState` ?

Reply:
I was going to call it `SharedState`. `MutableState` also works.